src/yadism/coefficient_functions/coupling_constants.py

```diff
@@ -92,9 +110,14 @@ def leptonic_coupling(self, mode, quark_coupling_type):
         # load Z coupling
         projectile_v = 0.0
         projectile_a = 0.0
-        if mode in ["phZ", "ZZ"]:
+        if mode in ["phZ", "ZZ", "Z4F"]:
             projectile_v = self.vectorial_coupling(abs(projectile_pid))
             projectile_a = self.weak_isospin_3[abs(projectile_pid)]
+            projectile_BSM_v = self.BSM_coupling_vectorial(abs(projectile_pid))
+            projectile_BSM_a = self.BSM_coupling_axial[abs(projectile_pid)]
+        if mode in ["ph4F"]:
+            projectile_BSM_v = self.BSM_coupling_vectorial(abs(projectile_pid))
```

Comment:
```suggestion
            projectile_BSM_v = self.BSM_coupling_vectorial[abs(projectile_pid)]
```

Reply:
Makes sense yes! I had forgotten to push this my bad